_posts/2017-11-15-Danse-macabre.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Danse Macabre"
+date:   2017-11-15
+source: Xanathar.153
+tags: [warlock, wizard, level5, necromancy]
+---
+
+**5th-level Necromancy**
```

Comment:
Necromancy should be lowercase. See the style guide in the root project directory.